src/modules/airspeed_selector/airspeed_selector_main.cpp

```diff
@@ -386,6 +386,9 @@ void AirspeedModule::update_params()
 {
 	updateParams();
 
+	float airspeed_stall = 7.0f;
+	param_get(param_find("FW_AIRSPD_STALL"), &airspeed_stall);
```

Comment:
@sfuhrer Let's check if we can use ModuleParams. And it not let's at least check if the param_update succeeded, if not then you can set the magic number. Like this you don't really know what airspeed_stall is set to after the call.

Reply:
64019907d170ca7e0c7cd8f8292c1941c978f8fc